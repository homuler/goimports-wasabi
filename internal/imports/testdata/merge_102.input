// merge comments
package foo

import/* prelparen comment */(/* postlparen comment */)
// footer comment

import
// prelparen comment
( // postlparen comment

	// bottom comment
) // postrparen comment
