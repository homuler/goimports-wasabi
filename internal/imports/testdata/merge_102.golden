// merge comments
package foo

import /* prelparen comment */  // prelparen comment
(      /* postlparen comment */ // postlparen comment
// bottom comment
) // postrparen comment
// footer comment
